packages/orchestration/src/examples/sendAnywhere.flows.js

```diff
@@ -33,7 +32,9 @@ export async function sendIt(
   // NOTE the proposal shape ensures that the `give` is a single asset
   const { give } = seat.getProposal();
   const [[_kw, amt]] = entries(give);
-  const { denom } = await findBrandInVBank(amt.brand);
+  const agoric = await orch.getChain('agoric');
+  const assets = await agoric.getVBankAssets();
+  const { denom } = NonNullish(assets.find(a => a.brand === amt.brand));
```

Comment:
+1 to moving lookup to the caller. Please include a message for helpful diagnostics when the band isn't found. https://github.com/Agoric/agoric-sdk/blob/e5f1d9b5b7bcaee731400c36b41b945fd42e7674/packages/internal/src/errors.js#L6-L10